sli/slistartup.cc

```diff
@@ -511,6 +519,7 @@ SLIStartup::init( SLIInterpreter* i )
   exitcodes->insert( exitcode_skipped_no_threading_name, Token( new IntegerDatum( EXITCODE_SKIPPED_NO_THREADING ) ) );
   exitcodes->insert( exitcode_skipped_no_gsl_name, Token( new IntegerDatum( EXITCODE_SKIPPED_NO_GSL ) ) );
   exitcodes->insert( exitcode_skipped_no_music_name, Token( new IntegerDatum( EXITCODE_SKIPPED_NO_MUSIC ) ) );
+  exitcodes->insert( exitcode_skipped_no_arborbackend_name, Token( new IntegerDatum( EXITCODE_SKIPPED_NO_ARBORBACKEND ) ) );
```

Comment:
In addition to @jougs suggestion this fixes the clang-formatting.

```suggestion
  exitcodes->insert(
    exitcode_skipped_no_recordingbackend_arbor_name, Token( new IntegerDatum( EXITCODE_SKIPPED_NO_RECORDINGBACKEND_ARBOR ) ) );
```